hubzero/hubzero-cms
# [PURR][#2524]award number to project creation graphic user interface and admin graphic user interface

https://purr.purdue.edu/support/ticket/2524

The award number is not part of the project grant information now. We want to add the award number as a field in project grant setting, so that researcher can provide it when they add or edit the grant information of the project. 

The changes are adding the award number text field in project creation graphic user interface and in admin graphic user interface.

Test:
1. Initialize a project. In third step, choose "yes" for question "IS THIS PROJECT SUPPORTED BY A RESEARCH GRANT?"
2. Fill out the grant information including the award number, and finish the project setup.
3. Go to the setting of the project that you just created and check if the award number is the same as you entered in the project creation procedure.
4. Open the project on admin interface and check if the award number is the same as you entered in the project creation procedure.

No hot fix needed.
